Description: Preferred user is distccd
 Starting the daemon as root causes it to change user, which
 should be to "distccd" in Debian.
Forwarded: not-needed
Author: Daniel Hartwig <mandyke@gmail.com>

--- a/source/src/setuid.c
+++ b/source/src/setuid.c
@@ -35,7 +35,7 @@
 #include "exitcode.h"
 
 
-const char *opt_user = "distcc";
+const char *opt_user = "distccd";
 
 
 /**
